src/buildblock/find_STIR_config.cxx

```diff
@@ -21,7 +22,7 @@ std::string find_STIR_config_file(const std::string& filename){
     return dir+"/"+filename;
 
 }
-
+//! The following finds the STIR configuration director and prints it out 
```

Comment:
```suggestion
//! The following finds the STIR configuration directory and prints it out 
```
However, it is much more useful to have a function that returns the string as opposed to printing it. Could be callable from Python then for instance